scanner/src/main/kotlin/provenance/NestedProvenance.kt

```diff
@@ -41,6 +39,15 @@ data class NestedProvenance(
      */
     val subRepositories: Map<String, RepositoryProvenance>
 ) {
+    /**
+     * Return the set of all contained [KnownProvenance]s.
+     */
+    val allProvenances: Set<KnownProvenance> =
```

Comment:
Does this need a `@JsonIgnore` annotation?

Reply:
Ouch, looks like it's indeed needed. Thanks.

Reply:
Commit message needs an update then.